src/NServiceBus.AcceptanceTests/Core/Recoverability/When_custom_policy_provided.cs

```diff
@@ -13,6 +13,8 @@ public class When_custom_policy_provided : NServiceBusAcceptanceTest
         [Test]
         public async Task Should_pass_recoverability_configuration()
         {
+            Requires.DelayedDelivery();
```

Comment:
Nitpick: As above we could potentially not make this test require delays

Reply:
We can, but it also seems like we'd be losing some coverage since in theory we could be properly passing  the configured immediate retry values, but not the delayed retry values.